docs/Configuration_usage.md

```diff
@@ -1,32 +1,55 @@
-| cache_dir                              | Point cache directory to other directory than default                                                                                             | ~/.foundry-dev-tools/cache             |
-| transforms_freeze_cache                | Uses offline cache as is; no Foundry connection needed                                                                                            | True or False (default: False)         |
-| transforms_output_folder               | When @transform in combination with TransformOutput.filesystem() is used, files are written to this folder.                                       | /projectA/local_files                  |
-| enable_runtime_token_providers         | Enables the token providers, Default is True                                                                                                      | True or False                          |
+| Name                                   | Description                                                                                                                                       | Values                                 | Default Values         |
+|----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------|------------------------|
+| foundry_url                            | Url of the foundry instance                                                                                                                       | e.g. https://foundry.example.com       | not set (None)         |
+| jwt                                    | The Bearer Token copied from the Foundry Token Page                                                                                               | eyJhb...                               | not set (None)         |
+| client_id                              | The client_id of the Foundry Third Party application for SSO Authentication                                                                       | ...                                    | not set (None)         |
+| client_secret                          | The client_secret of the Foundry Third Party application                                                                                          | ...                                    | not set (None)         |
+| grant_type                             | Set this to client_credentials if you use Foundry Third Party application client_credentials flow. Default is authorization_code                  | authorization_code, client_credentials | authorization_code     |
+| scopes                                 | Custom scopes for Third Party Application Tokens                                                                                                  | "api:read-data offline_access"         | not set (None)         |
```

Comment:
Looking at the code, I am not perfectly sure this will work as expected: Inside config, `scopes` is a string that contains a space-separated list if entries, while `foundry_api_client` seems to assume this to be a list of strings.

I could not find the `scopes.split(' ')` that I would have expected to see anywhere...